popSimManu.tex

```diff
@@ -505,7 +507,8 @@ \subsubsection*{ Single-population demographic models.}
 model it is also influenced by migration between the target population and populations not analyzed in inference.
 Since ancestral population sizes correspond to the inverse of the coalescence rates, by computing the
 inverse coalescence rates from the simulated model, we obtain the best estimates
-one may hope to infer by analyzing a single population. We thus analytically compute inverse coalescence
+one may hope to infer by analyzing a single population with a method that seeks to match coalescence time distributions.
+. We thus analytically compute inverse coalescence
```

Comment:
Looks like a stray full stop at the start of the line here.

Reply:
Yup. fixed.